### ISSUE TITLE
Wont change image

### ISSUE THREAD

**Issue by lespionage** [viljamis/ResponsiveSlides.js#106]
Hi!
I am having some problem with this solution. My images are being beautifully rescaled and the random function works great. however the slider does not change image at all. I have been trying to change the settings severel times but no luck. This is my configuration.

In my header:

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script> 

<script src="<?php bloginfo('template_directory') ;?>/responsiveslides.min.js"></script>
    

<script type="text/javascript"> 
  jQuery(function () {
 jQuery(".rslides").responsiveSlides({
  auto: true,             // Boolean: Animate automatically, true or false
  speed: 1000,            // Integer: Speed of the transition, in milliseconds
  timeout: 1000,          // Integer: Time between slide transitions, in milliseconds
  pager: true,           // Boolean: Show pager, true or false
  nav: true,             // Boolean: Show navigation, true or false
  random: true,          // Boolean: Randomize the order of the slides, true or false
  pause: false,           // Boolean: Pause on hover, true or false
  pauseControls: true,   // Boolean: Pause when hovering controls, true or false
  prevText: "Previous",   // String: Text for the "previous" button
  nextText: "Next",       // String: Text for the "next" button
  maxwidth: "",           // Integer: Max-width of the slideshow, in pixels
  controls: "",           // Selector: Where controls should be appended to, default is after the 'ul'
  namespace: "rslides",   // String: change the default namespace used
  before: function(){},   // Function: Before callback
  after: function(){}     // Function: After callback
});
 });
</script>

In my HTML

<ul class="rslides">
    <li><img src="<?php bloginfo('template_directory') ;?>/images/grabber/image1.jpg" alt="" /></li>
    <li><img src="<?php bloginfo('template_directory') ;?>/images/grabber/image2.jpg" alt="" /></li>
</ul>

My css

.rslides {
  position: relative;
  list-style: none;
  overflow: hidden;
  width: 100%;
  padding: 0;
  margin: 0;
  }

.rslides li {
  position: absolute;
  display: none;
  width: 100%;
  left: 0;
  top: 0;
  }

.rslides li:first-child {
  position: relative;
  display: block;
  float: left;
  }

.rslides img {
  display: block;
  height: auto;
  float: left;
  width: 100%;
  border: 0;
  }

In my directory i have 

responsiveslides.min.js
responsiveslides.js
responsiveslides.css

The first image is showing and when i reload the second one is randomized. But there is no slide.

/ Jonas
